packages/fluentui/react-bindings/src/hooks/useUnhandledProps.ts

```diff
@@ -0,0 +1,19 @@
+import getUnhandledProps from '../utils/getUnhandledProps';
+
+/**
+ * Returns an object consisting of props beyond the scope of the Component.
+ * Useful for getting and spreading unknown props from the user.
+ *
+ * @param handledProps - An array with names of props
+ * @param props - A ReactElement props object
+ * @returns A shallow copy of the prop object
+ */
+function useUnhandledProps<P extends Record<string, any>>(handledProps: (keyof P)[], props: P): Partial<P> {
```

Comment:
Can we add example in the react-bindings README? Or do you consider this as an internal hook?

Reply:
Added 👍